packages/react/src/test/create-component_test.tsx

```diff
@@ -490,17 +487,15 @@ suite('createComponent', () => {
     el.fire('foo');
```

Comment:
The line above this can be update to either no have the explicit `undefined`, or the comment above it can mention that both explicit `undefined` or omitting property will clear listener.